### PR TITLE
[ci][api][webui] Enable the Lint/Eval cop in rubocop

### PR DESCRIPTION
**Pull request by bgeuken**
This will remove some code evaluation in development mode which reads
in a .rb file.
This seems to be old unused code anyway. Besides it's a nasty hack.

### CODE REVIEW

File: src/api/config/environments/development.rb [hunk 1]
```diff
@@ -59,12 +59,3 @@
...
 CONFIG['frontend_protocol'] = "http"
-
-require 'socket'
-fname = "#{Rails.root}/config/environments/development.#{Socket.gethostname}.rb"
-if File.exist? fname
-  STDERR.puts "Using local environment #{fname}"
-  eval File.read(fname)
-else
-  STDERR.puts "Custom development.#{Socket.gethostname}.rb not found - using defaults"
-end
```
Comment (bgeuken) [inline src/api/config/environments/development.rb]: @adrianschroeter @coolo Would be good if you could double check that this isn't used anymore.

I checked our OBS instance and my personal dev machine and couldn't find a config/development.#{Socket.gethostname}.rb file.

(There is a config/development.sphinx.conf in my vagrant box, but that's for a different purpose, right?)

File: src/api/.rubocop.yml [hunk 2]
```diff
@@ -92,6 +92,9 @@ Lint/AssignmentInCondition:
 Lint/EndAlignment:
   AlignWith: variable
 
+Lint/Eval:
```
Comment (Ana06) [inline src/api/.rubocop.yml]: Could you add a comment up to this line explaining what the cop does?. I think those comments may be useful. Something like:

`# checks for the use of *Kernel#eval* as it is a serious security risk.`

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

**Comment by Ana06** [comment]
[Enable the Lint/Eval cop in rubocop](https://trello.com/c/gNBOgrZy/57-enable-the-lint-eval-cop-in-rubocop)

File: src/api/.rubocop.yml (hunk 2 above)
Comment (bgeuken) [inline src/api/.rubocop.yml]: Thought it's self explaining. I've changed it now.

Comment (Ana06) [inline src/api/.rubocop.yml]: But it is not like that for all cops and to be consistent I would add it for all of them. It is not a lot work and it may be helpful. :slightly_smiling_face:

File: src/api/config/environments/development.rb (hunk 1 above)
Comment (hennevogel) [inline src/api/config/environments/development.rb]: I often use it to set temporary config changes without. We have used it in the past for vagrant specific settings. I know @coolo uses it too somehow. Why do you want to remove it?

Comment (bgeuken) [inline src/api/config/environments/development.rb]: It's a hack. Besides, how is this different to other temporary changes we don't want to commit?

Comment (hennevogel) [inline src/api/config/environments/development.rb]: The difference is that you can have them in an untracked file...

File: src/api/app/controllers/build_controller.rb [hunk 3]
```diff
@@ -187,6 +187,7 @@ def file
       fsize = regexp[1]
       logger.info "streaming #{path}"
 
+      # rubocop:disable Lint/EndAlignment
```
Comment (hennevogel) [inline src/api/app/controllers/build_controller.rb]: Unrelated change right?

File: src/api/config/environments/development.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/config/environments/development.rb]: I would prefer it at least. The other option would probably be an include. Let's talk tomorrow:-)

Comment (hennevogel) [inline src/api/config/environments/development.rb]: Okay then. remove it...

File: src/api/app/controllers/build_controller.rb (hunk 3 above)
Comment (bgeuken) [inline src/api/app/controllers/build_controller.rb]: Double checked it. Still need 7beda23a13e012ab369f
